mcp/src/utils/environment.ts

```diff
@@ -58,19 +67,19 @@ export class CourierMcpConfig {
       fileConfig['BASE_URL'] ||
       fileConfig['base_url'] ||
       'https://api.courier.com';
-    this.showLogs =
-      headers?.['SHOW_LOGS'] ??
-      headers?.['show_logs'] ??
-      fileConfig['SHOW_LOGS'] ??
-      fileConfig['show_logs'] ??
-      true;
+    this.logLevel = parseLogLevel(
```

Comment:
You might want to just convert these to lowercase or upper and then get the value if we want to support both. I think HTTP headers are generally case-insensitive